src/cf-pagination/src/cf-pagination.less

```diff
@@ -16,10 +16,8 @@
 
 // Sizing variables
 
-@pagination-font-size-px:  @base-font-size-px;
-@pagination-font-size-em:  unit( @pagination-font-size-px / @base-font-size-px, em );
-@pagination-btn-height-px: 42px;
-@pagination-btn-width-px:  130px;
+@pagination-font-size-px:     @base-font-size-px;
```

Comment:
This is only ever used when calculating layout dimensions, not for setting type sizing. Could we drop it in favor of using `@base-font-size-px` on it's own?